src/plots/cartesian/index.js

```diff
@@ -0,0 +1,21 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+
+'use strict';
+
+var Plotly = require('../../plotly');
+
+
+exports.type = 'cartesian';
```

Comment:
In the gist I had written up for the registry, I was using `name`. I think I prefer it to `type` because `type` can be a bit ambiguous as to whether we mean `cartesian` or `plot`.